### DISCUSSION TITLE
Why does the Ubuntu PPA include RCs? Shouldn't they be published on the unstable PPA?

### DISCUSSION THREAD

**Discussion by wjandrea** [strawberrymusicplayer/strawberry#1542] [Q&A] [answered]
I just ran an update and saw that version `1.1.2-rc1-focal` is being offered, but I'm not using [the unstable PPA](https://launchpad.net/~jonaski/+archive/ubuntu/strawberry-unstable), I'm using [the regular PPA](https://launchpad.net/~jonaski/+archive/ubuntu/strawberry). Shouldn't RCs be published on the unstable PPA? Or are they considered "stable enough" or something like that?

**Comment by jonaski** [comment, marked as answer]
I'm open to suggestions, but I think distributing it to PPA is a good idea for several reasons.
It's supposed to be stable, but of course that's not always the case, but not a lot of users are testing directly from git, so if I distribute the RC to the PPA there is a lot more chance a bug get's fixed before the final release which goes to all linux distributions. With the PPA I can easily and quickly publish a new RC with a fix, but if a really bad bug get's into a final release it goes into all Linux distros repositories.
If I don't publish the RC to the PPA, there are not many users that actually tests the RC so there is a lot less point having a RC at all.
But I could create a third PPA possibly called strawberry-stable which always has the latest stable release.

**Reply by wjandrea** [reply]
OK, I'm more confused after reading that. I did a bit of research and I noticed [the website](https://www.strawberrymusicplayer.org/#download) says *"Strawberry is available in the package repositories on most Linux distributions including ... Ubuntu"*, but it's not actually available on Ubuntu 20.04 (Focal). Is that the problem? I had thought the PPA was where you were publishing stable releases.

For other Ubuntu versions, which repo is it published to? universe, right? I have that enabled.

**Reply by jonaski** [reply]
That's right, Strawberry is not available in the official repositories on Ubuntu Focal. The first release of Strawberry was released in 2018 and Focal was released in 2020, which means they could have included it in time for the release.
However, Focal also has a very old Qt 5 version (Qt version 5.12.8) which is no longer supported, and there is no Qt 6 available in Focal, Strawberry 1.1.2 will be the last release that supports Qt 5, as supporting Qt 5 is holding the project back quite a bit, we can't make use of all new functionality in Qt 6, so I'm dropping support for all the old Ubuntu releases older than Noble.
If there is a critical bug, I can backport a fix to the PPA, but I won't do that for all bugs, as it's too much work maintaining multiple branches.

**Reply by wjandrea** [reply]
OK, so then I guess the website should be changed to say *"Linux distributions including ... Ubuntu (22.04+)"* or something along those lines.

Then regarding the PPA, if this is the last release for Focal, I guess it doesn't matter what you do with it.

thanks :)
